gordo_components/client/client.py

```diff
@@ -552,17 +574,20 @@ def predict(
             return PredictionResult(
                 name=endpoint.target_name, predictions=None, error_messages=[msg]
             )
+        else:
+            logger.info(f"Processing {start} -> {end}")
+            if isinstance(response, dict):
+                predictions = server_utils.dataframe_from_dict(response["data"])
+            else:
+                predictions = server_utils.dataframe_from_parquet_bytes(response)
```

Comment:
Note that these lines do the same as the lined 475-480, but slightly different

Reply:
Moved into a common place.